test/setup_test.rb

```diff
@@ -12,6 +12,7 @@ def test_it_exists
   end
 
   def test_it_can_collect_ships
+    skip
```

Comment:
We don't need this test anymore correct since we got rid of the collect_ships method?

Reply:
Good catch. I went back and removed it.